roles/vpn/templates/rules.v4.j2

```diff
@@ -1,3 +1,13 @@
+{% if ansible_product_name == 'Google Compute Engine' %}
+*mangle
+:PREROUTING ACCEPT [0:0]
+:INPUT ACCEPT [0:0]
+:FORWARD ACCEPT [0:0]
+:OUTPUT ACCEPT [0:0]
+:POSTROUTING ACCEPT [0:0]
+-A FORWARD -s {{ vpn_network }} -p tcp -m tcp --tcp-flags SYN,RST SYN -j TCPMSS --set-mss 1348
```

Comment:
As an alternative, an MSS value could be defined in the various `cloud-*` roles, then we could do away with the `ansible_product_name` conditional entirely.